USAGE.rst

```diff
@@ -39,26 +39,31 @@ To execute a build pipeline locally, try the following example command:
      python execute_pipeline.py \
        --pipeline_kwargs={\'sleep_secs\':2} SamplePipeline
 
-To execute a build pipeline remotely, start two consoles and run the following command
-in one console:
+Remote
+******
+
+To execute a build pipeline remotely, start two consoles (on the same or different machines) and run 
+the following command in one console:
 
   ::
 
-     python start_conductor.py
+     TOOLKIT_HOME=<path_to_toolkit_repo> python start_conductor.py \
+       --log_local --jobboard=<unique_jobboard_name>
 
 
-And then run the following command for multiple times with different parameter:
+And then run the following command in the other console with the same jobboard name:
 
   ::
 
      python execute_pipeline.py \
        --pipeline_kwargs={\'sleep_secs\':2} \
-       --env remote SamplePipeline
+       --env=<unique_jobboard_name> SamplePipeline
 
 Please note, there is no guarantee that your pipeline job will be claimed by the
-conductor running on your machine. It can also be picked up by other conductor
-processes. You can run the second command for multiple times, and chances are
-good that your conductor will pick up one job at least.
+conductor running on the first machine unless you choose a jobboard name that no one
+else is using. Otherwise, it can be picked up by other conductor processes.
+
+Be sure you are running Java 7. There might be issues with Java 8.
```

Comment:
Right, I was referring to which version of Java is in the PATH. I was seeing an issue when that was version 8 on my workstation, and I fixed that issue by switching to 7. Unfortunately, I can't reproduce the issue now, so I'm dropping this statement until someone sees it again.